.bonsai.yml

```diff
@@ -0,0 +1,58 @@
+---
+description: "Sensu CHANGEME"
+builds:
+- platform: "linux"
+  arch: "amd64"
+  asset_filename: "sensu-CHANGEME_#{version}_linux_amd64.tar.gz"
```

Comment:
ideally, `sensu-CHANGEME` would be a template variable, for repository/project name etc.